ckanext/example_iconfigurer/tests/test_example_iconfigurer.py

```diff
@@ -63,22 +63,22 @@ def test_build_extra_admin_nav_config_option_present_but_empty(self):
         response = app.get('/build_extra_admin_nav')
         nosetools.assert_equal(response.body, expected)
 
-    @helpers.change_config('ckan.admin_tabs', {'ckanext_myext_config_one': 'My Label'})
+    @helpers.change_config('ckan.admin_tabs', {'ckanext_myext_config_one': {'label': 'My Label', 'icon': None}})
     def test_build_extra_admin_nav_one_value_in_config(self):
         '''
         Correct string returned when ckan.admin_tabs option has single value in config.
         '''
         app = self._get_test_app()
-        expected = """<li><a href="/ckan-admin/myext_config_one"><i class="fa fa-picture-o"></i> My Label</a></li>"""
```

Comment:
Why this icon is not shown anymore? Does this mean, that existing portals may lose some icons as well?

Reply:
The add_ckan_admin_tab function is not used in ckan core and has no effect.
By default, when I use the "add_ckan_admin_tab" function in extension, the icon does not appear.
In test-case, ckan_icon is set in the mapper, and the icon comes out via named_route.
And as I said above, if I do not set the icon in the "add_ckan_admin_tab" function, I think it is correct to just display the label without the icon. ('icon': None)